JSDemos/Demos/SelectBox/GroupedItems/React/App.js

```diff
@@ -37,7 +39,8 @@ class App extends React.Component {
               dataSource={fromUngroupedData}
               valueExpr="ID"
               grouped={true}
-              displayExpr="Name" />
+              displayExpr="Name"
+              value={1} />
```

Comment:
We should use uncontrolled mode in React.
It's not possible now to change the value through UI
```suggestion
              defaultValue={1} />
```